nemo_rl/utils/logger.py

```diff
@@ -323,6 +329,73 @@ def log_plot(self, figure: plt.Figure, step: int, name: str) -> None:
         """
         self.run.log({name: figure}, step=step)
 
+class SwanlabLogger(LoggerInterface):
+    """Weights & Biases logger backend."""
+
+    def __init__(self, cfg: SwanlabConfig, log_dir: Optional[str] = None):
+        self.run = swanlab.init(**cfg)
+        print(
+            f"Initialized SwanlabLogger for project {cfg.get('project')}, run {cfg.get('name')} at {log_dir}"
+        )
```

Comment:
looks like swanlab has configuration for a[ local dir](https://docs.swanlab.cn/api/py-init.html#:~:text=and%20JSON%20files.-,logdir,-(str)%20Offline%20dashboard), should we be plumbing it in like this to 
```suggestion
    def __init__(self, cfg: SwanlabConfig, log_dir: Optional[str] = None):
        self.run = swanlab.init(**cfg, logdir=log_dir)
        print(
            f"Initialized SwanlabLogger for project {cfg.get('project')}, run {cfg.get('name')} (with offline logdir={log_dir})"
        )
```